frontend/dispatcher.lua

```diff
@@ -584,6 +584,11 @@ function Dispatcher:getNameFromItem(item, settings, dont_show_value)
     return title
 end
 
+function Dispatcher:getArgFromValue(item, value)
```

Comment:
On mobile, so i can't code search too well, but is this trying to reverse from the display string to the argument?

Do we store the translated string for the setting?

Reply:
https://github.com/koreader/koreader/blob/ccd29d36c0cde78d956bbd01b7232a9d70c81ffd/frontend/ui/data/koptoptions.lua#L86-L92
For exmple, `trim_page`.
It converts `values` (that is stored in `document.configurable.trim_page`) to `args` (that is expected by Dispatcher).

Reply:
Yes, the translated string (from `toggle`) is displayed in QuickMenu.